calypso/api.go

```diff
@@ -0,0 +1,193 @@
+package calypso
+
+import (
+	"github.com/dedis/cothority"
+	"github.com/dedis/cothority/byzcoin"
+	"github.com/dedis/cothority/darc"
+	"github.com/dedis/onet"
+	"github.com/dedis/protobuf"
+)
+
+// Client is a class to communicate to the calypso service.
+type Client struct {
+	byzcoin  *byzcoin.Client
+	onet     *onet.Client
```

Comment:
you don't need a onet.Client, just use the byzcoin Client (it should have SendProtobuf too). actually, it might be better if you just embed the byzcoin client to calypso. e.g.,
```
 type Client struct { 
  *byzcoin.Client
  ltsReply *CreateLTSReply
}

Reply:
I was thinking if someone wants to separate them and not use both on the same client, it may be good to allow for this, what do you think?

Reply:
We usually keep the features minimal until we know something is necessary for sure. Further, the Java API is quite similar - https://github.com/dedis/cothority/blob/master/external/java/src/main/java/ch/epfl/dedis/calypso/CalypsoRPC.java#L24 to what I described, the byzcoin client is the super class of the calypso client